src/app/file-manager/file-modal/file-modal.component.html

```diff
@@ -0,0 +1,19 @@
+<h2 mat-dialog-title>Where do you want to download to?</h2>
+
+<mat-dialog-content>
+  <div class="flex-container">
+      <div class="browse-button">
+          <label for="directorypicker">Browse...</label>
+          <input type="file" id="directorypicker"  (change)="directoryChanged($event)" style="display: none;" webkitdirectory directory multiple required/>
```

Comment:
When I tried this out I noticed that the Browse button worked but didn't have a visible response to a mouse click. Is there a way to fix that?

Reply:
can you please give us more details?

Reply:
Sure, the behavior I see is when I click on the Browse button, it doesn't change its visible state. I would expect it to become darker, like the Cancel button does when it's clicked on.

![image](https://user-images.githubusercontent.com/1799360/33144869-82c80998-cf8c-11e7-8ccf-e095a43660a6.png)

In this screen shot the mouse is clicking on the Cancel button, so it's gray, not white. When I click on the Browse... button it stays white. This has the effect of making it look like the button isn't being clicked on.